smart_contract/executor/src/permission.rs

```diff
@@ -322,15 +855,15 @@ impl<T: Permission> From<&T> for OnlyGenesis {
 pub(crate) fn accounts_permissions() -> impl Iterator<Item = (AccountId, PermissionObject)> {
     FindAllAccounts
         .execute()
-        .dbg_expect("failed to query all accounts")
+        .expect("INTERNAL BUG: `FindAllAccounts` must never fail")
```

Comment:
why you replaced `dbg_expect`?

Reply:
i'm afraid that default panic won't print anything in `wasm`

Reply:
sure, I'll restore it